pkgs/applications/networking/instant-messengers/nchat/default.nix

```diff
@@ -30,10 +32,6 @@ stdenv.mkDerivation rec {
     zlib
   ] ++ lib.optionals stdenv.isDarwin [ AppKit Cocoa Foundation ];
 
-  cmakeFlags = [
-    "-DHAS_WHATSAPP=OFF" # go module build required
```

Comment:
Why? It's still ON by default https://github.com/d99kris/nchat/blob/15b08991bfd8ac10440934691a036bd4a959f4dd/CMakeLists.txt#L46

Reply:
It is turned off automatically after not finding go https://github.com/d99kris/nchat/blob/15b08991bfd8ac10440934691a036bd4a959f4dd/CMakeLists.txt#L61